Mathlib/Data/Finset/Image.lean

```diff
@@ -884,6 +884,21 @@ theorem finsetCongr_toEmbedding (e : α ≃ β) :
 
 end Equiv
 
+section InjOn
+
+variable {α β : Type*} {f : α → β}  {s : Set α} (hs : Set.InjOn f s)
+
+/-- The image of a Finset under a map on which it is injective -/
+def Finset.map_of_injOn {s : Finset α} (hs : Set.InjOn f s) :
```

Comment:
This doesn't match the naming convention.

@YaelDillies, thoughts on this definition in general?

Reply:
Uh, I doubt we want this? Either you replace `Finset.map` everywhere or you simply use `Finset.image` in your application.

Note that `Finset.preimage` will soon change from assuming `s.InjOn f` to `s.FinitelyFiberedOn f`.

Reply:
This comes from https://leanprover.zulipchat.com/#narrow/stream/217875-Is-there-code-for-X.3F/topic/Finset.2EInjOn.2Emap/near/404409525

Reply:
In any case, subsequent modifications made it useless here, so I just deleted it.